[Term Entry] Python statsmodels: model residuals (#5903)
Codecademy/docs: content/python/concepts/statsmodels/terms/model-residuals/model-residuals.md

---
Title: 'Model Residuals'
Description: 'Analyzes residuals for Python statistical models, measure model performance, detect patterns, and diagnose problems using concise syntax and examples.'
Subjects:
  - 'AI'
  - 'Data Science'
  - 'Machine Learning'
Tags:
  - 'Data'
  - 'Linear Regression'
  - 'Logistic Regression'
  - 'Models'
  - 'Statsmodels'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

**Model residuals** are calculated as the differences between observed and predicted values for a statistical model. This method measures error or deviation for each data point by using the formula:

> ! [Model Residuals formula](https://raw.githubusercontent.com/Codecademy/docs/main/media/model-residual-example.png)

Reviewer: Why is there a space in between ! and [?

---
Title: 'Model Residuals'

> Description: 'Analyzes residuals for Python statistical models,measure model performance, detect patterns, and diagnose problems using concise syntax and examples.'

Reviewer: ```suggestion
Description: 'Analyzes residuals for Python statistical models, measure model performance, detect patterns, and diagnose problems using concise syntax and examples.'
```

Subjects:
  - 'AI'
  - 'Data Science'
  - 'Machine Learning'
Tags:
  - 'Data'
  - 'Linear Regression'
  - 'Logistic Regression'
  - 'Models'
  - 'Statsmodels'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

> **Model residuals** are calculated as the differences between observed and predicted values for a statistical model. This is a method of error or deviation measuring for each data point by using the formula:

Reviewer: ```suggestion
**Model residuals** are calculated as the differences between observed and predicted values for a statistical model. This method measures error or deviation for each data point by using the formula:
```

\[

> \text{Residual} = \text{Observed Value} - \text{Predicted Value}

Reviewer: Put an image of the formula here, rather than writing it down.

\]

> Residuals are a the key concept used in statistical modeling for evaluating the goodness of fit, identifying patterns, detecting outliers, and validating assumptions regarding the model. Analysis of residuals helps in enhancing model accuracy and reliability by providing information about areas where the model is underperforming.

Reviewer: ```suggestion
Residuals are a key concept in statistical modeling. They are used to evaluate the goodness of fit, identify patterns, detect outliers, and validate assumptions about the model. Analyzing residuals helps enhance model accuracy and reliability by providing information about areas where the model is underperforming.
```
